packages/playwright-ct-vue/index.d.ts

```diff
@@ -34,25 +34,22 @@ export type PlaywrightTestConfig = Omit<BasePlaywrightTestConfig, 'use'> & {
   }
 };
 
-interface MountResult<Props = { [key: string]: any }> extends Locator {
+export interface MountOptions<Props = Record<string, unknown>> {
+  props?: Props,
+  slots?: Record<string, unknown>,
+  on?: Record<string, Function>,
+  hooksConfig?: any,
+}
+
+interface MountResult<Props = Record<string, unknown>> extends Locator {
   unmount(): Promise<void>;
-  setProps(props: Partial<Props>): Promise<void>
+  rerender(options: { props: Props }): Promise<void>
```

Comment:
We are nearing the branch point, so we should probably call this `_rerender` for the time being, until we make it work for jsx + react + svelte.